### PR TITLE
Feature/ecasogp 6817

### PR DESCRIPTION
**Pull request by santoshGSA**

### CODE REVIEW

File: _layouts/video-card.html [hunk 1]
```diff
@@ -26,6 +61,38 @@
                   <a  class="text-black ogp-card__link--unset" href="{{ site.baseurl }}{{single_video.directory}}{{single_video.page_name}}">
                       <b>{{single_video.title}} </b>
                   </a>
+                  <button type="button" class="usa-button usa-button--outline" style="float:right;" id="btnShare">Share</button>
```
Comment (alexdntowe) [inline _layouts/video-card.html:64]: please remove style and use custom class or uswds utilities classes

File: _layouts/video-card.html [hunk 2]
```diff
@@ -26,6 +61,38 @@
...
+                  <ul class="usa-button-group usa-button-group--segmented" style="float:right;display: none;" id="shareLinkUl">
+                    <li class="usa-button-group__item">
```
Comment (alexdntowe) [inline _layouts/video-card.html:66]: please remove style and use custom class or uswds utilities classes

**Review by alexdntowe — COMMENTED** [review]
If ti can't be dont this sprint, please add a clean up task in the technical story to separate the js code from the html code

File: _layouts/video-card.html (hunk 1 above)
Comment (santoshGSA) [reply _layouts/video-card.html]: used uswds class

File: _layouts/video-card.html (hunk 2 above)
Comment (santoshGSA) [reply _layouts/video-card.html:32]: used uswds class

**Review by alexdntowe — APPROVED** [review]

**Pull request merged**